tools-v2/pkg/cli/command/curvebs/query/snapshot/snapshot.go

```diff
@@ -0,0 +1,164 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-12-04
+ * Author: ZackSoul
+ */
+package snapshot
+
+import (
+	"strconv"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	snapshotutil "github.com/opencurve/curve/tools-v2/internal/utils/snapshot"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	listSnapshot "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/snapshot"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+const (
+	snapshotExample = `$ curve bs query snapshot --path /test/test111 --snapshotstatus done 
+(other status: "in-progress" "deleting" "errorDeleteing" "canceling" "error")`
```

Comment:
Sorry, maybe I didn't make myself clear, please keep the README and the example consistent, all use `#`. this is the last suggestion.

Reply:
Understand, that has been corrected.